src/Microsoft.AspNetCore.Authentication.Google/GoogleHandler.cs

```diff
@@ -100,8 +104,11 @@ protected override string BuildChallengeUrl(AuthenticationProperties properties,
             return authorizationEndpoint;
         }
 
-        private static void AddQueryString(IDictionary<string, string> queryStrings, AuthenticationProperties properties,
-            string name, string defaultValue = null)
+        private static void AddQueryString(
```

Comment:
I break the lines which are longer than 120 characters. 

By the way, in other repos we try to avoid default parameter. Do we do it here?